tests/integration/dualstack/dualstack_int_test.go

```diff
@@ -19,7 +19,7 @@ var dualStackServerArgs = []string{
 	"--disable-network-policy",
 }
 var _ = BeforeSuite(func() {
-	if !testutil.IsExistingServer() && os.Getenv("CI") != "" {
+	if !testutil.IsExistingServer() && os.Getenv("CI") != "true" {
```

Comment:
maybe use https://pkg.go.dev/strconv#ParseBool instead of requiring literal "true"?

Reply:
Converting to that makes the if statement quite complex, because if a "" is passed, you have to check the error, not the bool value
```
if ci, err := strconv.ParseBool(os.Getenv("CI")); err != nil && !testutil.IsExistingServer() && !ci {
```
also, Github guarentees that it will be "true"
https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables